client/lib/analytics/test/utils.js

```diff
@@ -0,0 +1,34 @@
+/** @format */
+/**
+ * External dependencies
+ */
+
+import { expect } from 'chai';
+
+/**
+ * Internal dependencies
+ */
+import { shouldReportOmitBlogId } from '../utils';
+
+describe( '#shouldReportOmitBlogId', () => {
+	test( 'should allow blog_id reporting for site-specific paths', () => {
+		expect( shouldReportOmitBlogId( '/stats/day/example.wordpress.com' ) ).to.be.false;
+		expect( shouldReportOmitBlogId( '/pages/example.wordpress.com' ) ).to.be.false;
```

Comment:
After using jest, `.to.be.` turns into `.toBe`

https://facebook.github.io/jest/docs/en/expect.html